Abot/Crawler/WebCrawler.cs

```diff
@@ -710,6 +712,42 @@ protected virtual void ProcessPage(PageToCrawl pageToCrawl)
             }
         }
 
+        private void ProcessRedirect(CrawledPage crawledPage)
```

Comment:
This code is heavily based on the content of the SchedulePageLinks method. Maybe it'd be worth to rename this one to ScheduleRedirect, for example.